src/ophys_etl/decrosstalk/active_traces.py

```diff
@@ -0,0 +1,465 @@
+import numpy as np
```

Comment:
Can you remind me where this module came from? I'm wondering if it should be imported from a separate repo rather than included in this repo, as it doesn't really conform to the rest of the code and might be hard to maintain.

Reply:
The file was cut-and-pasted from here

https://github.com/AllenInstitute/mesoscope_ica/blob/main/mesoscope_ica/active_traces.py

(I then reformatted it to get it to pass the linter, but I have not changed any logic)

Based on a conversation I had with Natalia, it's a combination of code written by Allen researchers and outside researchers. It's pretty fine tuned (to the point where, even though the 3 in the denominator on line 134 cannot be justified from first principles, we did not want to change it because the code is tuned to give the correct results for real data). My long term goal is to replace it with code from the event detection pipeline Dan is writing, once that is mature enough to use here.